src/quickstart/provider/azure-acs/boot.md

```diff
@@ -0,0 +1,200 @@
+# Booting Azure Container Service
+
+## Prerequisites
+
+1. Azure Account - If you do not already have a Azure Cloud account, you can start a trial with $200 of free credit [here](https://azure.microsoft.com/en-us/free/). After completing sign up, you must add your billing information.
+2. Some form of *nix-based terminal - MacOS, Ubuntu, CentOS, Bash on Windows, etc
+<br>Where the following is present:
```

Comment:
linebreaks in numbered lists? You can use the following format for a numbered list within a numbered list if that's what you'd like to achieve here:

1. foo
  a. bar
2. car
3. star